patientsearch/jsonify_abort.py

```diff
@@ -0,0 +1,9 @@
+"""Utility function to bring together flask `abort` and `jsonify`"""
+from flask import abort, jsonify, make_response
+
+def jsonify_abort(**kwargs):
+    """Takes any number of args, like jsonify, but raises like abort with correct content_type heading"""
+    status_code = kwargs.pop('status_code', 500)
```

Comment:
Might be nice to have `status_code` as a proper keyword arg (can be defaulted to `None`/`"500"`), so it's visible in `help(jsonify_abort)` etc

Reply:
👍